schemas/expconf/v0/s3.json

```diff
@@ -42,6 +42,14 @@
             ],
             "default": null
         },
+        "prefix": {
+            "type": [
+                "string",
+                "null"
+            ],
+            "pattern": "^[^.]*(?:\\.[^.]+)*$",
```

Comment:
Yeah, I totally forgot about stuff like `asdf..asdf` and also, `blah/.../stuff` <- which I guess is legal. I like your suggestion way better.

Reply:
Also, you test with `grep -E` doesn't work because the regexes here have to be string escaped first. E.g., matching `.` in `grep -E` would be `\.` but here it must be `\\.` because there is escape char overlap.